README.md

```diff
@@ -1 +1,142 @@
-# budget_app
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run Linters](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [ImproveLife] <a name="about-project"></a>
+
+> The ImproveLife app helps you manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+**[ImproveLife]** is a ruby on rails mobile website.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> ImproveLife is backend project built with:
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Ruby on Rails</summary>
+</details>
+
+<details>
+<summary>HTML/CSS</summary>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> In this app you can do the following things:
+
+- register and log in, so that the data is private to them.
+- introduce new transactions associated with a category.
+- see the money spent on each category.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [ImproveLife](https://budget-app-cnpg.onrender.com)
```

Comment:
- [ ] You did well with implementing this change, however, this live demo link that you provided still does not work and it leads to a "Time out". Below I will include a screenshot of it for you.

![image](https://user-images.githubusercontent.com/86318284/225262775-5b7ab557-fe99-4610-b71a-38b3284b8dee.png)

**I have done some due diligence and checked it in both private mode and even asked a friend of mine to confirm it for me and it does not work in all cases.**

If for some reason, it loads perfectly fine on your end, kindly include a screencast clearly showing the link as well in it, and kindly perform sign up, sign in, and create a group and its activities to proof that it does actually work.

[Here](https://youtu.be/8ofyx5slcug) is an awesome video by a Micronaut showing how you can deploy your app to render.

Reply:
Hello @Nemwel-Boniface !
Thanks for your review. Indeed you were right the deployment was having some issues but I finally managed to complete it. So I updated the deployed app link in the readme file and here is a loom video link of the deployed app in case the link doesn't work: https://www.loom.com/share/8cd40b56723249c5a357a5f8ecb9f41b